calc/src/Components/Buttons.js

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import Input from "./Input";
+import "../Styles/Buttons.css";
+
+class Buttons extends React.Component {
```

Comment:
Because this component doesn't track state, think about updating to be a functional component